_posts/2018-07-17-client-openings.md

```diff
@@ -0,0 +1,20 @@
+---
+layout: post
+author: chris
+title: Let us help!
+image: 'posts/diagram.jpg'
+---
+
+Apsis Labs is about to wrap up our largest contract yet, and has openings for new clients! [Contact us](mailto:contact@apsis.io) to tell us about your project.
+
+Apsis is a small custom software consultancy. That means we help you use software to solve your business problems. We specialize in mobile and web applications in the healthtech, business workflow, and game industries. We're flexible, fast, and easy to work with.
+
+We'd love to build something for you. Got this far and still not sure you need us? Here are some good clues we could help you out:
+
+ * You've got a great app idea for your business or soon-to-be business, but no idea how to turn the idea into a reality.
+ * You've spent hours of your valuable time moving data in and between spreadsheets or documents to track your business information.
+ * You've got your own software engineers, but your backlog is full of killer features you'll never have time to implement.
+ * You've hired, or are thinking about hiring, people to manually enter data from external sources into your systems.
+ * Your employees spend hours doing repetitive work that is important to you, but is not a core part of your business.
+
+If any of the above are true or you just think you might need a software developer's help, give us a shout ([contact@apsis.io](mailto:contact@apsis.io)). Consulting with us is free - we will happily take a look at your project and either work up an estimate or point you to one of our partners who'd be a great fit!
```

Comment:
"Consulting with us is free" I kind of want to find a way to re-phrase this comment ("Contacting us is free", "Starting the process is free", something like that).

I just could imagine that people would describe us as "Consultants", and so if we say "Consulting with us is free" it might make it seem like...I don't know. It makes the overall service feel cheaper to me for some reason.

Dunno if anyone else agrees. Feel free to disregard, as this is just a one-off subjective interpretation, and it might be that no one else reads it the same way.

Reply:
"We won't bill you until we start your project. The initial consultation is free."

?

Reply:
I think just "Initial consultation" is enough to make my reaction go away entirely.